lib/config/pacote.js

```diff
@@ -46,9 +46,17 @@ function pacoteOpts (moreOpts) {
     Object.assign(opts, ownerStats)
   }
 
+  if (!!npm.config.get('always-auth')) {
+    if (!opts.auth) { opts.auth = {} }
+    opts.auth.alwaysAuth = true;
+    if (!!npm.config.get('_auth')) {
+      opts.auth._auth=npm.config.get('_auth');
+    }
+  }
+
   npm.config.keys.forEach(function (k) {
     const authMatch = k[0] === '/' && k.match(
-      /(.*):(_authToken|username|_password|password|email|always-auth)$/
+      /(.*):(_authToken|username|_password|password|email|always-auth|_auth)$/
```

Comment:
So, looked at this issue and here is where I stand:
- PR created for pacote. I think that should clear. Once that lands we can update the dependency in npm
- There is only one general test for config/pacote in npm, so the first thing I need to do is create a test for config parsing
- Fix config parsing so we get all options (including _auth) both on global and scoped.